app/data_providers/dispatch.py

```diff
@@ -137,7 +147,15 @@ async def get_slate_lineup(
         else:
             slates += [self.recommended_reads_slate_provider.get_slate()]
 
-        slates += [self.collection_slate_provider.get_slate()]
+        if contentv1_assignment is not None and contentv1_assignment.variant == 'treatment':
```

Comment:
will be able to see somewhere that it didn't go to the experiment branch more often than we expect? maybe we should log what branch was used or add some telemetry?

Reply:
- This PR includes [telemetry for the variant and name](https://github.com/Pocket/recommendation-api/pull/917/files#diff-877dd70fd79464930f75be17e641f0d44fa1d51a17c4d659f39b730f0f7b5905R79-R88) in the same event that emits the slate lineup. This makes it easy to join between slate metadata and the experiment.
- [HS-347](https://getpocket.atlassian.net/browse/HS-347) In a future PR we'll emit a [variant_enroll](https://console.snowplowanalytics.com/cf0fba6b-23b3-49a0-9d79-7ce18b8f9618/data-structures/1945896af520f129eb62b20ceef8cca5be4e3285754b7e51093c089a1ccc7ee9) event that's required by the analytics team. Their experiment dashboard shows the split.

Reply:
ok, I just want to make sure we'll be able to see it if there's a bug somewhere and contentv1_assignment is None or the variant name is wrong